.github/workflows/manual-update-cli-docs.yaml

```diff
@@ -0,0 +1,50 @@
+name: Manually retrigger the update of the OCM CLI documentation
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: Which version (e.g. v0.42.0) do you want to publish?
+        required: true
+        default: ''
+
+jobs:
+  retrigger:
+    name: Create new "Release Publish Event"
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      id-token: write
+      packages: write
+    steps:
+    - name: Generate token
+      id: generate_token
+      uses: tibdex/github-app-token@v2
+      with:
+        app_id: ${{ secrets.OCMBOT_APP_ID }}
+        private_key: ${{ secrets.OCMBOT_PRIV_KEY }}
+    - name: Ensure proper version
+      run: |
+        if [ -z "${{ github.event.inputs.version }}" ]; then
+          echo "Version not provided"
+          exit 1
+        fi
```

Comment:
I am not sure about the importance of this check. We only check, that the variable is not empty. There is no version-validation or the like.

Thus, I would assume that the following `grep` would offer the same functionality (by failing, if it does not find the release which it won't because the variable is empty). Therefore, I would delete this check.

Additionally, I am wondering if the `required: true` would make sure that the variable must be set

Reply:
you're right, empty strings `    ` are not accepted or seem to be truncated automatically by github